### PR TITLE
Feature/limit submodule

### PR DESCRIPTION
**Pull request by Shreyash-work-em**
The shared models migration repo has been removed as a git submodule in this branch.
It has been added as a separate directory. 
References changed to accommodate the submodules are maintained
This is done so that we can put to test the work without having the deployment error as a roadblock.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -15,20 +15,19 @@
 from sqlalchemy.pool import NullPool
 
 
-
+# Random comment
```
Comment (karntrehan) [inline app.py]: Delete this please.

File: app.py [hunk 2]
```diff
@@ -109,15 +106,15 @@ async def dmp_updates():
 
         # Loop through all dmp issues
         dmp_tickets = await DmpCronQueries.get_all_dmp_issues(async_session)
-        
+
```
Comment (karntrehan) [inline app.py:106]: Lets undo format changes.

File: shared_migrations/db/discord-bot.py [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+from sqlalchemy.orm import sessionmaker
+from .models import *
+from sqlalchemy.ext.declarative import DeclarativeMeta
+from sqlalchemy.ext.asyncio import create_async_engine, AsyncSession
+
+# load_dotenv()
+
+class DiscordBotQueries:
+    # def __init__(self):
+    #     DB_HOST = os.getenv('POSTGRES_DB_HOST')
```
Comment (karntrehan) [inline shared_migrations/db/discord-bot.py]: Remove comments?

File: app.py (hunk 2 above)
Comment (Shreyash-work-em) [reply app.py:106]: Rolled back to original app.py

**Pull request merged**